src/Cache.ts

```diff
@@ -2,6 +2,7 @@ import debug from 'debug';
 import envPaths from 'env-paths';
 import * as fs from 'fs-extra';
 import * as path from 'path';
+import { format, parse } from 'url';
```

Comment:
```suggestion
import * as url from 'url';
```

Named imports that are super generic in name can be confusing.  "format what", "parse what".  I prefer to keep the `url` namespace similar to the `fs` and `path` imports above